src/auth/is-auth.service.ts

```diff
@@ -69,7 +74,10 @@ export class IsAuthService {
   }
 
   private async verifyAccessToken(accessToken: string) {
-    if (!accessToken) throw new Unauthorized('Access token not provided');
```

Comment:
Avoid throwing an exeception as this will reject the request right there and will not check a maybe perfectly valid long lived refresh token